cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -425,7 +425,9 @@ def test_vm_power_options_from_off(provider, soft_assert, testing_vm, ensure_vm_
     check_power_options(provider, soft_assert, testing_vm, testing_vm.STATE_OFF)
 
 
-@pytest.mark.uncollectif(lambda provider: not provider.one_of(VMwareProvider))
+@pytest.mark.uncollectif(lambda provider: not provider.one_of((VMwareProvider, RHEVMProvider)))
+@pytest.mark.meta(blockers=[BZ(1571830, forced_streams=['5.8', '5.9'],
```

Comment:
Just wondering if it'd be easier to read this if this was taken care of in 1 lambda:

```python
@pytest.mark.uncollectif(
     lambda provider: (provider.one_of(RHEVMProvider) and BZ(1571830, forced_streams=['5.8', '5.9']).blocks)
         or not provider.one_of((RHEVMProvider, VMwareProvider))
)
```

Reply:
Better to have separate BZ blocker, as we split the reasons why we skip the test